### PR TITLE
meta: update dates for 4.x EOL and 6.x maintenance

### PR DESCRIPTION
**Pull request by MylesBorins**
I've updated the schedule.json and the README.md

Not sure the best way to update schedule.ical, how do we create it?

There is a part of me that thinks we should remove it and put the dates
on the official calendar. I'll open another PR for that

Closes: https://github.com/nodejs/Release/issues/264

### CODE REVIEW

**Comment by MylesBorins** [comment]
I'm throwing this on the agenda so was can assure we have signoff on the new dates

**Comment by MylesBorins** [comment]
updated based on conversation in #264 PTAL

File: README.md [hunk 1]
```diff
@@ -6,9 +6,9 @@
 | :--:    | :---:               | :---:      | :---:            | :---:                 | :---:                     |
 | v0.10.x | **End-of-Life**     | -          | -                | 2015-10-01            | 2016-10-31                |
 | v0.12.x | **End-of-Life**     | -          | -                | 2016-04-01            | 2016-12-31                |
-| [4.x][] | **Maintenance LTS** | [Argon][]  | 2015-10-01       | 2017-04-01            | April 2018                |
+| [4.x][] | **Maintenance LTS** | [Argon][]  | 2015-10-01       | 2017-04-01            | 2017-04-31                |
```
Comment (gibfahn) [inline README.md]: Don't think April 31 exists 😉

**Comment by MylesBorins** [comment]
I could have sworn I got that right. I checked my knuckles and everything

**Comment by xiaoyu860519** [comment]
！

File: README.md (hunk 1 above)
Comment (gyandeeps) [reply README.md]: Also i think it should be 2018. :smile_cat:

**Pull request merged**